README.md

```diff
@@ -57,12 +57,12 @@ You can also leverage on alpine distributions to create smaller runtime images:
 
 ```Dockerfile
 ### base image ###
-FROM hmcts.azurecr.io/hmcts/base/node/alpine-lts-10 as base
+FROM hmctspublic.azurecr.io/base/node/alpine-lts-10:10-alpine as base
 COPY package.json yarn.lock ./
 RUN yarn install --production
 
 ### build image (Debian) ###
-FROM hmcts.azurecr.io/hmcts/base/node/stretch-slim-lts-10 as build
+FROM hmctspublic.azurecr.io/base/node/stretch-slim-lts-10:10-stretch-slim as build
```

Comment:
I'm a bit confused on why there's a repo for everything and a specific tag that matches the same repo name...

Reply:
It's the "generic" tag that mirrors what's in the base (imported) image. To avoid using latest. Btw I don't think the docs PR is really the right place for this conversation...

Reply:
hah sure, just asking, it confused me yesterday...

Reply:
Alternatively, they could be:
base/node/stretch-slim-lts:10-stretch-slim
base/node/stretch-slim-lts:8-stretch-slim
and
base/node/alpine-lts:8-alpine
base/node/alpine-lts:10-alpine
?
That's probably better ... but still this is not the right place for this conversation :-)